agent/04_agent_configure.sh

```diff
@@ -122,21 +122,7 @@ metadata:
   namespace: cluster0
 spec:
   baseDomain: ${BASE_DOMAIN}
-  clusterInstallRef:
```

Comment:
This is definitely used: https://github.com/openshift/assisted-service/blob/da8164f04040feae88de132de4cfe02447129dde/internal/controller/controllers/clusterdeployments_controller.go#L140-L146